packages/cli-e2e/utils/cli.ts

```diff
@@ -9,6 +9,16 @@ export function killCliProcess(cliProcess: ChildProcessWithoutNullStreams) {
   return waitForKill;
 }
 
+export function killCliProcessFamily(
+  cliProcessLeader: ChildProcessWithoutNullStreams
+) {
+  const waitForKill = new Promise<void>((resolve) => {
+    cliProcessLeader.on('close', () => resolve());
+  });
+  process.kill(-cliProcessLeader.pid);
```

Comment:
As explained above, this command kill and process within the same group.

Reply:
🥇 nice finding!